.github/workflows/web_build.yml

```
@@ -0,0 +1,105 @@
+      name: ${{ steps.info.outputs.name }}
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+      - name: Fetch tags
+        run: git fetch --prune --unshallow --tags
+      - name: Get info
+        id: info
+        # The tag name should be retrieved lazily, as tagging may be delayed.
+        env:
+          BRANCH: ${{ github.event.workflow_run.head_branch }}
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2236:style:4:9: Use -n instead of ! -z

(shellcheck)